routers/cities-router.js

```diff
@@ -28,26 +28,41 @@ const Cities = require('../models/cities-model.js');
 //     })
 // })
 
+
+
+
+
+// router.get('/', async (req, res) => {
+//     const cities = await Cities.findAllWinter()
+//     try {
+//         res.status(200).json(cities)
+//     }
+//     catch (err) {
+//         res.status(500).json({ message: 'Error while getting city', err })
+//     }
+// })
+
 router.get('/', async (req, res) => {
-  const cities = await Cities.findAllWinter()
-  try {
-    res.status(200).json(cities)
-  }
-  catch (err) {
-    res.status(500).json({ message: 'Error while getting city', err })
-  }
+    const cities = await Cities.findAllWinterAndSummer()
```

Comment:
is this getting all data that frontend need? Doesn't look like this function has data from location table